e2e/testcases/pod-template/risingwave-spot-toleration.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: risingwave.singularity-data.com/v1alpha1
+kind: RisingWavePodTemplate
+metadata:
+  name: spot-toleration
+template:
+  metadata:
+    labels:
+      risingwave/spot: "1"
```

Comment:
why the value is "1"?

Reply:
It's just an example. The point is to use the pod template.